specification/dashboard/resource-manager/readme.md

```diff
@@ -0,0 +1,36 @@
+# Dashboard
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Dashboard.
+
+---
+## Getting Started
+To build the SDK for Dashboard, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+---
+## Configuration
+### Basic Information
+These are the global settings for the Dashboard API.
+
+```yaml
+title: DashboardManagementClient
+openapi-type: arm
+openapi-subtype: rpaas
+tag: package-2021-09-01-preview
+```
+
+### Tag: package-2021-09-01-preview
+
+These settings apply only when `--tag=package-2021-09-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-09-01-preview'
+input-file:
+    - Microsoft.Dashboard/preview/2021-09-01-preview/grafana.json
+```
```

Comment:
Please add a section `Swagger to SDK` in readme.md. An example can be found here: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/compute/resource-manager/readme.md#swagger-to-sdk